pygmo/docstrings.cpp

```diff
@@ -2526,6 +2526,61 @@ used for Tchebycheff and boundary interception decomposition methods.
 )";
 }
 
+std::string unconstrain_docstring()
+{
+    return R"(__init__(udp, method, weights = [])
+
+The unconstrain meta-problem.
+
+This meta-problem transforms a constrained problem into an unconstrained problem applying one of the following methods:
+
+* Death penalty: simply penalizes all objectives by the same high value if the fitness vector is infeasible.
+* Kuri's death penalty: defined by Angel Kuri Morales et al., penalizes all objectives according to the rate of satisfied constraints.
+* Weighted violations penalty: penalizes all objectives by the weighted sum of the constraint violations.
+* Ignore the constraints: simply ignores the constraints.
+* Ignore the objectives: ignores the objectives and defines as a new single objective the overall constraints violation (i.e. the sum of the L2 norms of the equalities and inequalities violations)
+
+**NOTE** The use of :class:`~pygmo.core.unconstrain` discards gradients and hessians so that if the original user defined problem
+implements them, they will not be available in the unconstrained problem. The reason for this behaviour is that,
+in general, the methods implemented may not be differentiable. Also, the use of this class was originally intended for
+derivative-free optimization.
+
+See: Coello Coello, C. A. (2002). Theoretical and numerical constraint-handling techniques used with evolutionary algorithms: 
+a survey of the state of the art. Computer methods in applied mechanics and engineering, 191(11), 1245-1287.
+
+See: Kuri Morales, A. and Quezada, C.C. A Universal eclectic genetic algorithm for constrained optimization,
+Proceedings 6th European Congress on Intelligent Techniques & Soft Computing, EUFIT'98, 518-522, 1998.
+
+The constructor admits two forms:
+
+* no arguments,
+* two mandatory arguments and one optional arguments.
+
+Any other combination of arguments will raise an exception.
+
+Args:
+    udp: a user-defined problem (either C++ or Python - note that *udp* will be deep-copied
+      and stored inside the :class:`~pygmo.core.unconstrained` instance)
+    method (``str``): a string containing the unconstrain method chosen, one of [``'death penalty'``, ``'kuri'``, ``'weighted'``, ``'ignore_c'``, ``'ignore_o'``]
```

Comment:
I think the visualization is clearer like this as it separates with commas keywords